src/components/rmrk/Gallery/AvailableActions.vue

```diff
@@ -176,6 +180,33 @@ export default class AvailableActions extends Mixins(RmrkVersionMixin) {
 
   }
 
+  protected async isTransactionPending() {
+    try {
+      const nft = await this.$apollo.query<{ nfts: PendingTransaction[] }>({
+        client: 'rmrkApolloClient',
+        query: verificationGuard,
+        variables: {
+          id: this.nftId
+        },
+      })
```

Comment:
What does this do? first time seing this weird syntax

Reply:
> weird syntax

by default is `query` function of type `T` and you can say to it what's the expected return type